geonetwork/microservices/searching/gn-cloud-searching-application.yml

```diff
@@ -0,0 +1,90 @@
+server:
+  instance-id: ${spring.application.name}:${spring.application.instance_id:${spring.cloud.client.ip-address}}:${server.port}
+spring:
+  rabbitmq.host: localhost
+  cloud:
+    config:
+      discovery:
+        enabled: false
+      enabled: false
+management:
+  health.ldap.enabled: false
+eureka:
+  client:
+    enabled: false
+    registerWithEureka: false
+    fetch-registry: false
+
+gn:
+  datadir: ${java.io.tmpdir}/gn-datadir
+  baseurl: https://georchestra.mydomain.org/geonetwork
```

Comment:
Can the FQDN be obtained using a var ?

Reply:
dunno i've just copypasted it from `default.properties` ;)

Reply:
i doubt the java standalone services has any knowledge of `default.properties` anyway since all those new monorepo-separated-microservices are decoupled from each other..

@pmauduit ?

Reply:
so amend `gn-cloud-searching.service` systemd service file do add this new property ? sigh :)

Reply:
we can do that, or have the previously cited yaml file templated in the ansible recipe (as this is what we already do for other services), i don't really mind, what do you prefer ? In any way, having ansible in mind, we already have this `fqdn` variable in the playbook.

Reply:
sure im not worried by the ansible case, but that PR is about the datadir repo which doesnt have variables, and @fvanderbiest was concerned about getting the FQDN from a variable :)

Reply:
the targeted branch (master) already assumes that the georchestra will be available behind georchestra.mydomain.org, I don't have any solutions to inject the default.proprties file from the datadir unfortunately, as the microservice come from upstream, part for forking once again. Maybe @groldan will have an idea ?